<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wBugDoc</class>
 <widget class="QMainWindow" name="wBugDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wBugDoc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ressources.qrc">
    <normaloff>:/rec/data/icons/iconBug.svg</normaloff>:/rec/data/icons/iconBug.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QTableView" name="tvwBugs">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnAdd">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../ressources.qrc">
              <normaloff>:/rec/data/icons/iconAdd.svg</normaloff>:/rec/data/icons/iconAdd.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelete">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../ressources.qrc">
              <normaloff>:/rec/data/icons/iconDelete.svg</normaloff>:/rec/data/icons/iconDelete.svg</iconset>
            </property>
            <property name="shortcut">
             <string notr="true">Del</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,2,0,0,0,0,0">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="lblID">
          <property name="text">
           <string notr="true">ID:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ledID">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblTitle">
          <property name="text">
           <string notr="true">Title:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ledTitle"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblDescription">
          <property name="text">
           <string notr="true">Description:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPlainTextEdit" name="pteDescription">
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblLocation">
          <property name="text">
           <string notr="true">Location:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="ledLocation"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblPicturePath">
          <property name="text">
           <string notr="true">Picture path:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="ledPicturePath">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>8</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblPicture">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>533</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblUnsavedChanges">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red</string>
        </property>
        <property name="text">
         <string notr="true">Unsaved changes.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnWebDiskCopy">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../ressources.qrc">
            <normaloff>:/rec/data/icons/iconFbh.png</normaloff>:/rec/data/icons/iconFbh.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveEntry">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string notr="true">Save entry</string>
        </property>
        <property name="shortcut">
         <string notr="true">Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnWait">
        <property name="text">
         <string notr="true">Wait (ALT + B)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pgbProgress">
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBugDoc">
    <property name="title">
     <string notr="true">Window</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionBack_to_home"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string notr="true">Export</string>
    </property>
    <addaction name="actionHTML"/>
   </widget>
   <widget class="QMenu" name="menuSpecials">
    <property name="title">
     <string notr="true">Specials</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuBugDoc"/>
   <addaction name="menuSpecials"/>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconOpen.svg</normaloff>:/rec/data/icons/iconOpen.svg</iconset>
   </property>
   <property name="text">
    <string notr="true">Load BugDoc</string>
   </property>
   <property name="iconText">
    <string notr="true">Load BugDoc</string>
   </property>
   <property name="toolTip">
    <string notr="true">Load BugDoc</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="actionBack_to_home">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconHome.svg</normaloff>:/rec/data/icons/iconHome.svg</iconset>
   </property>
   <property name="text">
    <string notr="true">Back to home</string>
   </property>
   <property name="iconText">
    <string notr="true">Back to home</string>
   </property>
   <property name="toolTip">
    <string notr="true">Back to home</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconQuit.svg</normaloff>:/rec/data/icons/iconQuit.svg</iconset>
   </property>
   <property name="text">
    <string notr="true">Close</string>
   </property>
   <property name="iconText">
    <string notr="true">Close</string>
   </property>
   <property name="toolTip">
    <string notr="true">Close</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHTML">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconDownloadFile.svg</normaloff>:/rec/data/icons/iconDownloadFile.svg</iconset>
   </property>
   <property name="text">
    <string notr="true">HTML</string>
   </property>
   <property name="iconText">
    <string notr="true">HTML</string>
   </property>
   <property name="toolTip">
    <string notr="true">HTML</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../../ressources.qrc">
     <normaloff>:/rec/data/icons/iconPreferences.svg</normaloff>:/rec/data/icons/iconPreferences.svg</iconset>
   </property>
   <property name="text">
    <string notr="true">Preferences...</string>
   </property>
   <property name="iconText">
    <string notr="true">Preferences</string>
   </property>
   <property name="toolTip">
    <string notr="true">Preferences</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
